release_contacts_from_quarantine.R

```diff
@@ -0,0 +1,95 @@
+library(tidyverse)
+library(dotenv)
+library(REDCapR)
+library(lubridate)
+library(sendmailR)
+
+source("functions.R")
+
+script_name <- "release_contacts_from_quarantine.R"
+script_run_time <- with_tz(now(), tzone = Sys.getenv("TIME_ZONE"))
+
+Sys.getenv("INSTANCE")
+
+read_contacts_data <- read_redcap_db(redcap_uri = Sys.getenv("URI"),
+                                     token = Sys.getenv("CONTACT_TOKEN"),
+                                     guess_max = 3000,
+                                     project_title = Sys.getenv("CONTACT_PROJECT_TITLE"))
+
+if (length(read_contacts_data) > 2) {
+
+ contacts_data <- read_contacts_data$data
+
+ candidates_for_release <- contacts_data %>%
+  filter(contact_released_from_quarantine == 0 &
+           contact_quarantine_date <= as_date(script_run_time)) %>%
+  select(record_id, redcap_event_name, contact_released_from_quarantine,
+         contact_quarantine_date, cleared_for_campus_timestamp, case_emp_ufid)
```

Comment:
```suggestion
 candidates_for_release <- contacts_data %>%
  filter(contact_released_from_quarantine == 0 &
           contact_quarantine_date <= as_date(script_run_time) &
           contact_method_prefer %in% c(1,3)) %>%
  select(record_id, redcap_event_name, contact_name_first, contact_name_last,
         contact_released_from_quarantine, contact_quarantine_date, contact_released_from_quarantine_date,
         cleared_for_campus, cleared_for_campus_timestamp,
         contact_method_prefer_sms, contact_method_prefer_email,
         case_emp_ufid)
```

I was missing one criterium. Also I added a bunch of columns that make the debugging/verification easier. They are inconsequential after this step as only record_id is consumed.